### PR TITLE
sale_delivery_note: new module for delivery notes

### PR DESCRIPTION
**Pull request by nuria-opusvl**
Allow printing of delivery note from delivery order screen.
Delivery note accessible from Sales Order screen.

Please review :-)

### CODE REVIEW

**Comment by pedrobaeza** [comment]
I think this module has a more general approach than yours: https://github.com/odoomrp/odoomrp-wip/tree/8.0/sale_documents_comments. Please check it, and if you like it, we'll propose it to OCA (it's in the roadmap, but not as the most important, so we can accelerate it).

**Comment by nuria-opusvl** [comment]
Pedro,

I have just installed your module, thank you.

It's a different functionality, though. A delivery note is basically the delivery details that needs to be printed out and stuck in the package that's about to go out. It's not like a customer's comment, it's information about the delivery itself.

deliverynoteexample

**Comment by max3903** [comment]
CLA is OK

**Comment by nuria-opusvl** [comment]
Hi,

Is the module ok in spite of the build failure?

Ta,
Nuria

**Comment by pedrobaeza** [comment]
If you rebase, you should now get a green Travis

**Comment by nuria-opusvl** [comment]
Thank you Pedro, we've just rebased.

**Comment by nuria-opusvl** [comment]
Hi, what else do I need to do?

**Comment by elicoidal** [comment]
Merging into the repos needs to be done by the reviewer (with at least 3 
positive reviews AFAIK).
We need to avoid "automatic" merging (and in this case the committer is 
the reviewer)

**Comment by nuria-opusvl** [comment]
Sorry mate, I am dyslexic, I read green to merge and I merged, I do 
apologise, I will try to do better next time :-)

On 08/09/15 10:07, Eric Caudal wrote:

> Merging into the repos needs to be done by the reviewer (with at least 3
> positive reviews AFAIK).
> We need to avoid "automatic" merging (and in this case the committer is
> the reviewer)
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/OCA/sale-workflow/pull/194#issuecomment-138488365.

## 

N. Arranz-Velazquez
OpusVL Odoo Specialist Team (OOST)
Product Owner

OpusVL
Drury House
Drury Lane
Rugby
CV21 3DE

T: 01788 298 450
W: www.opusvl.com

**Comment by Nick-OpusVL** [comment]
Hi.

I've taken a look at jobs 878.9 and 878.10 here:
- https://travis-ci.org/OCA/sale-workflow/builds/77136608

It looks like it's complaining that it can't find a module called sale_transport_multi_address, even though the OCA repository stock-logistics-transport is declared in the oca_dependencies.txt file.

I've looked at the commits by @nuria-opusvl and neither of them touch that file, and the sale_delivery_note module being contributed doesn't depend on that module.

I think I've found the culprit in Job 878.7:

```
$ git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools

Cloning into '/home/travis/maintainer-quality-tools'...

fatal: unable to access 'https://github.com/OCA/maintainer-quality-tools.git/': gnutls_handshake() failed: A TLS packet with unexpected length was received.

The command "git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools" failed and exited with 128 during .
```

Which looks like a transient TLS (infrastructure) related error rather than a code error.

Is there some way to prod Travis to try the build again?

Nick

**Comment by nuria-opusvl** [comment]
Hi can anyone run the build again, please?

**Comment by rafaelbn** [comment]
Hi @nuria-opusvl , It's complicated to get reviewers as you see... and now in v8 dificult. But I have rebuild bots, let's see if all is green :smile:

File: sale_delivery_note/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+
+{
+    'name': 'Delivery Notes',
+    'version': '0.1',
+    'author': 'OpusVL, Odoo Community Association (OCA)',
+    'website': 'http://opusvl.com/',
```
Comment (nikul-serpentcs) [inline sale_delivery_note/__openerp__.py:28]: The website should be `https://github.com/OCA/sale-workflow`

**Comment by rousseldenis** [comment]
@nuria-opusvl Do you plan to finish this ?

**Comment by rousseldenis** [comment]
Closing this as no activity. @nuria-opusvl Please feel free to reopen it if you still need it

**Pull request closed without merging**